pkg/apis/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package apis contains API Schema definitions for the various API groups
+package apis
```

Comment:
Does this need an annotation similar to those in `pod/doc.go` and `run/v1alpha1/doc.go`?

Reply:
It doesn't strictly need one, but we could add `+k8s:deepcopy-gen=package` like in https://github.com/knative/serving/blob/main/pkg/apis/doc.go#L17. I don't think it would actually do anything, though.